iXBRLViewerPlugin/viewer/src/html/inspector.html

```diff
@@ -86,6 +86,15 @@ <h3 data-i18n="inspector.filter">Filter</h3>
             <option value="summationOrContributor" data-i18n="inspector.summationOrContributor">Summation or Contributor</option>
           </select>
 
+          <div class="control-label" data-i18n="inspector.dimensions">
```

Comment:
Pre-existing, it would be nice if the search order options matched the order of the attributes in the fact panel.

Reply:
@sagesmith-wf I can address this in XT-2959 if you weren't already planning on it. That PR will be touching all of these search elements anyway.

Reply:
Going to have Aaron address this in his ticket